### PR TITLE
added 2 countries

### PR DESCRIPTION
**Pull request by bltemrah**

### CODE REVIEW

File: src/main/java/org/example/countries.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Country
+
+1. Japan
+2. Italy
```
Comment (CevriyeTurkmen) [inline src/main/java/org/example/countries.txt:4]: It shouldnt be Italy, as I remember it must be Germany.

File: src/main/java/org/example/countries.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Country
+
+1. Japan
```
Comment (dilekdeniz) [inline src/main/java/org/example/countries.txt:3]: the first option should be England

**Review by dilekdeniz — APPROVED** [review]

**Review by CevriyeTurkmen — APPROVED** [review]

**Review by syftzr — APPROVED** [review]

File: src/main/java/org/example/countries.txt (hunk 1 above)
Comment (bltemrah) [reply src/main/java/org/example/countries.txt]: done

File: src/main/java/org/example/countries.txt (hunk 2 above)
Comment (bltemrah) [reply src/main/java/org/example/countries.txt]: done

**Review by syftzr — COMMENTED** [review]
It's ok.

**Pull request merged**